plugins/org.yakindu.sct.generator.c/src/org/yakindu/sct/generator/c/StatemachineHeader.xtend

```diff
@@ -123,6 +124,7 @@ class StatemachineHeader implements IContentTemplate {
 		/*! Enumeration of all states */ 
 		typedef enum
 		{
+			NOT_INITIALIZED_STATE, /* State with value 0 is the initial value of the StateConfVector */
```

Comment:
- The enum value does not follow the existing naming pattern. Also the 'last state' value uses the stjatemachine name prefix to avoid name clashes.
- the state should be named something like 'XYZ_NO_STATE' 
- Is there any objection to replace the 'lastState' value by the 'no state' (value 0 instead of #states)